.github/workflows/dependabot.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     - name: Set up Go 1.x
       uses: actions/setup-go@v2
       with:
-        go-version: '1.16'
+        go-version: '1.17'
```

Comment:
Q: are we still using dependant or can we drop this file entirely (not blocking for this PR)

Reply:
Afaik it's currently not running. I can drop it in a follow-up PR when we have consensus to drop it.
(/cc @randomvariable )